pkg/controller/plan/kubevirt.go

```diff
@@ -183,10 +189,15 @@ func (r *KubeVirt) buildImport(vm *plan.VMStatus) (object *vmio.VirtualMachineIm
 		err = liberr.Wrap(err)
 		return
 	}
+	err = r.Builder.Resolve(&vm.Ref)
```

Comment:
The Builder.Resolve() will do an inventory lookup and ensure all of the fields (both ID and Name) are populated.  This is needed for naming created CRs and other actions to follow.